homeassistant/components/shelly/switch.py

```diff
@@ -30,19 +22,6 @@
 )
 
 
-@dataclass(frozen=True)
-class BlockSwitchDescription(BlockEntityDescription, SwitchEntityDescription):
-    """Class to describe a BLOCK switch."""
-
-
-GAS_VALVE_SWITCH = BlockSwitchDescription(
```

Comment:
This was added before 2024.1 cycle started right? We should deprecate this properly before removing it. I would suggest creating an issue when someone uses the switch, checkout harmony switch for an example.

Reply:
Good point, thanks.